doc/src/overrides.md

```diff
@@ -77,9 +77,10 @@ case for nightly-only software that pins to a revision from the release
 archives.
 
 In these cases the toolchain can be named in the project's directory in a file
-called `rust-toolchain.toml` or `rust-toolchain`. If both files are present in
-a directory, the latter is used for backwards compatibility. The files use the
-[TOML] format and have the following layout:
+called `.rust-toolchain.toml`, `rust-toolchain.toml` or `rust-toolchain`.
+If multiple files are present in a directory, the latter present one is used
```

Comment:
"the latter in the list" could clarify since there are more than 2 now.

Reply:
Maybe it would be better to use an enumeration for the precedence of the files.

Reply:
Did the enumeration.

Reply:
No worries at all! Thanks for working on this. :)